test/fetch.js

```diff
@@ -16,3 +16,16 @@ test.serial('relative URLs are passed to fetch unresolved', async t => {
 	t.is(await ky('unicorn', {prefixUrl: '/api/'}).text(), '/api/unicorn');
 	global.fetch = originalFetch;
 });
+
+test.serial('fetch option takes a custom fetch function', async t => {
+	const customFetch = async input => {
+		return new Response(input.url);
+	};
```

Comment:
```suggestion
	const customFetch = async input => new Response(input.url);
```

Reply:
I think we should add some assertions inside of the `customFetch` function and maybe use `t.plan()` to test that the function actually gets called the expected number of times.

So at the top of the test, we would add something like `t.plan(7)` and then in the `customFetch` function:

```js
const customFetch = async (input) => {
	t.is(input instanceof Request);
	return new Response(input.url);
};
```

I should have done this in the other test, too.

Reply:
added two more assertions now